test/index_test.js

```diff
@@ -593,7 +593,7 @@ suite('Project Config', () => {
   suite('json validation', () => {
     test('throws good error messages', () => {
       try {
-        ProjectConfig.validateAndCreate({ lint: [] });
+        ProjectConfig.validateAndCreate({lint: []});
```

Comment:
We don't usually format test files, can you undo this change? Maybe our `npm format` is misconfigured to include it?

Reply:
Probably. Updated!